spacy/pipeline/trainable_pipe.pyx

```diff
@@ -266,6 +265,10 @@ cdef class TrainablePipe(Pipe):
         """
         raise NotImplementedError(Errors.E931.format(parent="Pipe", method="add_label", name=self.name))
 
+    @property
+    def is_distillable(self) -> bool:
+        return not (self.__class__.distill is TrainablePipe.distill and self.__class__.get_teacher_student_loss is TrainablePipe.get_teacher_student_loss)
+
```

Comment:
I think this is a fine solution - maybe we can just add a comment here to explain what's going on, basically that we return `False` if there's no non-default implementation?

Reply:
Done!